processing/data_collection/gazette/spiders/rn_mossoro.py

```diff
@@ -0,0 +1,63 @@
+from datetime import datetime, date
+
+import dateutil
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RnMossoroSpider(BaseGazetteSpider):
+
+    name = "rn_mossoro"
+    allowed_domains = ["jom.prefeiturademossoro.com.br"]
+
+    TERRITORY_ID = "2408003"
+
+    def __init__(self, start_date=None, end_date=None, *args, **kwargs):
+        self.start_date = date.today().replace(day=1)
+        self.end_date = date.today()
+
+        super(RnMossoroSpider, self).__init__(start_date, end_date, *args, **kwargs)
+
+        self.logger.debug(
+            "Start date is {date}".format(date=self.start_date.isoformat())
+        )
+        self.logger.debug("End date is {date}".format(date=self.end_date.isoformat()))
+
+    def start_requests(self):
+
+        for year in range(self.start_date.year, self.end_date.year + 1):
+            for month in range(self.start_date.month, self.end_date.month + 1):
+                base_url = f"http://jom.prefeiturademossoro.com.br/{year}/{month}/"
+                yield scrapy.Request(base_url)
+
+    def parse(self, response):
+        for entry in response.css("article.post.category-jom"):
```

Comment:
Using the classes in the selector doesn't seem necessary at a first glance. What do you think?

Reply:
Indeed. All the `<article />` tags here are gazettes, but IMO the more specific, the better.
They they add other "entities" to the page, that are also rendered  `<article />`, this will still work.

Reply:
As a disclaimer, I think that this discussion isn't too relevant. So I'll be trying to keep it short and this would never block your PR. I'll just wait for your closing thoughts on this.

Generic selectors are not always good for the reason you gave. But, the more specific is not always the better since you introduce more points of failure.

In this particular case, since we are introducing specificity based on classes and the element tag is an "article" (which is not super common), I feel like specifying is not bringing much and opening doors to errors with class changes.

But leaving it like that is good too. It's still simple and relying on just a few things ;)